app.rb

```diff
@@ -0,0 +1,146 @@
+require './book'
+require './rental'
+require './person'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def menu
+    puts 'Please choose an option by enterin a number:'
+    puts "
+    1 - List all Books\n
+    2 - List all people\n
+    3 - Create a Person\n
+    4 - Create Book\n
+    5 - Create a rental\n
+    6 - List all Rentals for a given person id\n
+    7 - Exit\n"
+  end
+
+  def option_menu(option)
+    actions = {
+      1 => :books_all,
+      2 => :people_all,
+      3 => :new_person,
+      4 => :new_book,
+      5 => :create_rental,
+      6 => :list_rentals,
+      7 => :quit_app
+    }
+    action = actions[option]
+    if action.nil?
+      puts "Invalid option: #{option}"
+    else
+      send(action)
+    end
+    main
+  end
+
+  def quit_app
+    print 'thank you for using this app!'
+    exit
+  end
+
+  def people_all
+    if @people.empty?
+      puts 'No people found'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    end
+  end
+
+  def books_all
+    if @books.empty?
+      puts 'No books found'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+  end
```

Comment:
- [ ] Great job working so far but I think you have a problem here after creating a new book it is created successfully but when I try to display all books I cab't see the new books that I added it is just  show me "No Books found" so kindly make sure to write code to display it correctly ✨🔥


![image](https://user-images.githubusercontent.com/58553711/209344040-4b2106fa-dc37-43a1-a1e1-e86238390b54.png)

Reply:
i fix the showing books and people